packages/utils/src/ColorUtils.ts

```diff
@@ -89,30 +102,42 @@ class ColorUtils {
    *
    * e.g. `rgb(255, 255, 255)` -> `{ r: 255, g: 255, b: 255, a: 1 }`
    * e.g. `rgba(255, 255, 255, 0.5)` -> `{ r: 255, g: 255, b: 255, a: 0.5 }`
+   * e.g. `color(srgb 1 1 0 / 0.25)` -> `{ r: 255, g: 255, b: 0, a: 0.25 }`
    * @param rgbOrRgbaString The rgb or rgba string to parse
    */
   static parseRgba(
     rgbOrRgbaString: string
   ): { r: number; g: number; b: number; a: number } | null {
-    const [, name, args] = /^(rgba?)\((.*?)\)$/.exec(rgbOrRgbaString) ?? [];
-    if (name == null) {
-      return null;
-    }
+    // if color(srgb) format, we handle that differently
 
-    // Split on spaces, commas, and slashes. Note that this more permissive than
-    // the CSS spec in that slashes should only be used to delimit the alpha value
-    // (e.g. r g b / a), but this would match r/g/b/a. It also would match a mixed
-    // delimiter case (e.g. r,g b,a). This seems like a reasonable tradeoff for the
-    // complexity that would be added to enforce the full spec.
-    const tokens = args.split(/[ ,/]/).filter(Boolean);
+    const [r, g, b, a] =
+      rgbOrRgbaString.match(
+        // loose with the regex to allow for different formats between browsers
+        // take the first 4 digits that look like numbers, including decimals
+        // We've already checked it's a valid color with CSS.supports
+        /(?:\b\d+\.\d*|\b\d+|\.\d+)/g
+      ) ?? [];
 
-    if (tokens.length < 3) {
+    if (r == null || g == null || b == null) {
       return null;
     }
 
-    const [r, g, b, a = 1] = tokens.map(Number);
+    if (rgbOrRgbaString.startsWith('color(srgb')) {
+      return {
+        r: Math.round(Number(r) * 255),
+        g: Math.round(Number(g) * 255),
+        b: Math.round(Number(b) * 255),
+        a: Number(a ?? 1),
```

Comment:
Does this need to handle the > 1 scenerio like the one below?

Reply:
no, all values passed into the color(srgb) format are 0-1. Technically neither do, as we've tested css.supports elsewhere, but since this is a util I figured more safety around it wouldn't be a bad thing.

Reply:
I mean if we really wanted to be particular, we would clamp everything to 0-255 only as well. I think you had a test with a non-alpha value, so I did this to fix it.